clients/fides-js/__tests__/lib/cookie.test.ts

```diff
@@ -249,6 +257,10 @@ describe("saveFidesCookie", () => {
     { url: "https://www.another.com", expected: "another.com" },
     { url: "https://privacy.bigco.ca", expected: "bigco.ca" },
     { url: "https://privacy.subdomain.example.org", expected: "example.org" },
+    {
+      url: "https://privacy.subdomain.example.co.uk",
+      expected: "example.co.uk",
+    },
```

Comment:
Just for completeness, I'd add one or two other examples like:
```suggestion
    {
      url: "https://privacy.subdomain.example.co.uk",
      expected: "example.co.uk",
    },
    {
      url: "https://example.co.in",
      expected: "example.com.in",
    },
    {
      url: "https://example.co.jp",
      expected: "example.co.jp",
    },
```

It's pretty duplicative in the code today, yes, but I think it feels right